packages/realm/package.json

```diff
@@ -71,5 +76,12 @@
     "type": "git",
     "url": "https://github.com/realm/realm-js.git",
     "directory": "packages/realm"
+  },
+  "binary": {
+    "module_name": "realm",
+    "module_path": "generated/ts",
+    "host": "https://static.realm.io",
+    "package_name": "realm-v{version}-napi-v6-{platform}-{arch}.tar.gz",
+    "remote_path": "realm-js-prebuilds/{version}"
```

Comment:
Because we're leaving out the `api_versions` from this object, we signal to cmake-js that we're not targeting napi explicitly, which ensures we get the uv.h in our include path.

Reply:
> Just curious how this avoids importing uv.h
@takameyer this is where the magic happens 🪄

Reply:
Thank you  🤩